plugins/environment-variables-plugin.js

```diff
@@ -13,9 +13,14 @@ if (!(env === 'development' || env === 'production')) {
   throw new Error(`Invalid NODE_ENV loaded: ${env}.`);
 }
 const prefix = load('ROUTE_PREFIX', '');
-const assetPath = '/_static';
+assert(!prefix.endsWith('/'), 'ROUTE_PREFIX must not end with /');
+const assetPath = load('FRAMEWORK_STATIC_ASSET_PATH', `${prefix}/_static`);
```

Comment:
As an aside, we now prefix the `assetPath` with the `routePrefix` as opposed to concatenating them when necessary.  I don't think we ever want the asset path sans prefix so this should be fine, but wanted to point it out.